source/reference/cutover-process.txt

```diff
@@ -115,11 +132,16 @@ Steps
          :language: json
          :copyable: false
 
-      .. note::
+      After you submit a ``commit`` request, call the ``progress`` endpoint
+      to ensure that the ``mongosync`` state is ``COMMITTING`` or
+      ``COMMITTED``.
+
+      .. warning::
 
-         After you submit a ``commit`` request, call the ``progress`` endpoint
-         to ensure that the ``mongosync`` state is ``COMMITTING`` or
-         ``COMMITTED``.
+         If the source cluster shuts down before mongosync can commit, such
```

Comment:
can we make this a repeatable snippet so we don't have the same text copied over in 2 places?